source/NVDAObjects/UIA/edge.py

```diff
@@ -458,7 +458,7 @@ def _get_ariaProperties(self):
 	# in a list of strings like "something=true;current=date;". We want to capture one group, after the '='
 	# character and before the ';' character.
 	# This could be one of: True, "page", "step", "location", "date", "time"
-	RE_ARIA_CURRENT_PROP_VALUE = re.compile("current=(\w+);")
+	RE_ARIA_CURRENT_PROP_VALUE = re.compile("current=(?!false)(\w+);")

```

Comment:
This regex seems a bit broken:

1. RE_ARIA_CURRENT_PROP_VALUE.match("current=true;dummy=test;") properly matches
2. RE_ARIA_CURRENT_PROP_VALUE.match("var=val;current=true;dummy=test;") does not match

May be we need a unit test for this. cc @feerrenrut errenrut